dist/style/style-safari.css

```diff
@@ -2134,9 +2134,17 @@ h3.alpheios-inflections__credits-title {
 .alpheios-treebank {
   display: flex;
   flex-direction: column !important;
+  height: 100% !important;
 }
 .alpheios-treebank__frame {
+  /*
+  A minimal height of the treebank's iframe, shall be no less than 43em,
+  but no more than that because max width of panel's content is 800px and
+  treebank frame will not if in if too wide
+  */
+  width: 690px !important;
```

Comment:
the user will still be able to resize the panel though, correct?

Reply:
If other than treebank panels are visible, this element will be hidden and thus will have no effect on resizing. If a treebank panel is open, this will set the width of the inner panel only. The treebank panel itself can be resized and if it will become smaller than the inner panel, a horizontal scroll will appear. Is this a correct behavior? I'm not sure we can ever resize the inner panel with the tree dynamically since it's an iframe with dynamic content and we can assume nothing about what's inside.

Within a Safari stylesheet, we have those `!important` modifier everywhere and so far it seems there is no way around it 🙁.